examples/wines.get.json

```diff
@@ -1240,11 +1353,58 @@
                                     "locale": "it",
                                     "value": "Wine Spectator"
                                 }
+                            ],
+                            "sort_index": 30,
+                            "in_search": false
+                        },
+                        "year": 2016,
+                        "wine_year": 2013,
+                        "valuation": {
+                            "type": "points",
+                            "value": 91,
+                            "text": [
+                                {
+                                    "locale": "de",
+                                    "value": "91 Punkte"
+                                },
+                                {
+                                    "locale": "fr",
+                                    "value": "91 points"
+                                },
+                                {
+                                    "locale": "it",
+                                    "value": "91 punti"
+                                }
                             ]
                         },
-                        "year": 2013,
-                        "wine_year": 2011,
-                        "valuation": "91"
+                        "text": [
+                            {
+                                "locale": "de",
+                                "value": "91 Punkte Wine Spectator (Jahr 2016)"
+                            },
+                            {
+                                "locale": "fr",
+                                "value": "91 points Wine Spectator (an 2016)"
+                            },
+                            {
+                                "locale": "it",
+                                "value": "91 punti Wine Spectator (anno 2016)"
+                            }
+                        ]
+                    }
+                ],
+                "medals_text": [
```

Comment:
This wine has three medals, but only two for the given wine year. So only the texts of these two medals are combined here (wine spectator first, because the medal was awarded more recent).